src/modules/pass.js

```diff
@@ -245,9 +254,13 @@ PassFF.Pass = (function () {
               result.exitCode = -2;
               result.stderr = `The host app (v${version}) is outdated!`;
             } else if (result.exitCode !== 0) {
-              log.warn('Script execution failed',
-                result.exitCode, result.stderr, result.stdout);
-              PassFF.Page.notify("Script execution failed: \n" + result.stderr);
+              if (result.stderr.trim() == "Error: /otp is not in the password store.") {
```

Comment:
Wouldn’t it be better to do a check against the version of passff-host, like few lines earlier? Specially as it relies on a side-effect.

Reply:
I think the idea here is that the host app only really needs to be upgraded if you are using OTP functionality, once the required version gets bumped this could be removed. I'm also ok with just bumping the required version of the host app.

Reply:
I think @lanodan suggests replacing the `if (result.stderr.trim() == "...")` by `if (command == "otp" && semver.gt("1.1.0", version))`. This check should already be placed before the line 256 as additional `} else if ...`. By the way, please add "otp" to the list of supported commands in line 239.

Reply:
Thanks for the feedback!